Resources/input/fsh/ISiKTermin.fsh

```diff
@@ -19,6 +19,8 @@ Id: ISiKTermin
 * start 1..1 MS
 * end 1..1 MS
 * slot 0..* MS
+  * reference 1.. MS
+* slot ^comment = "Zur Referenzierung auf eine Slot-Ressource MUSS eine Reference.reference mit einer URL verwendet werden. Bei verteilten Systemen sollten absolute URLs benutzt werden. Auf dem gleichen Host sind auch kürzere, relative URLs zulässig." //Zur Begründung: verschiedene Referenzierungs-Arten (z.B. mit Business-Identifiern) sind ggf. nicht interoperabel
```

Comment:
Das Termin-Repository muss so gestaltet sein, dass aus Perspektive des Clients es nur eine Service-BaseUrl gibt. Im Falle eines verteilten Systems muss das Termin-Repository diese Komplexität verstecken und alles intern weiterleiten.

Reply:
Ist das dein Vorschlag für den Comment zum .slot? ... ich würde zumindest den Satz "Zur Referenzierung auf eine Slot-Ressource MUSS eine Reference.reference mit einer URL verwendet werden." vorangestellt stehenlassen... oder?

Reply:
Nein, kein Vorschlag nur eine Erläuterung. Ja, "Zur Referenzierung auf eine Slot-Ressource MUSS eine Reference.reference mit einer URL verwendet werden." können wir gern übernehmen.

Reply:
Vorschlag zur Umsetzung des Kommentars "Zur Referenzierung auf eine Slot-Ressource MUSS eine Reference.reference mit einer URL verwendet werden. Das Termin-Repository muss so gestaltet sein, dass es aus Perspektive des Clients nur eine Service-BaseUrl gibt." 

- [ ] zusätzlich muss die Klarstellung zur Client-Sicht in der Basis textuell (informativ) festgehalten werden. Ggf hier als erstes: https://simplifier.net/guide/Implementierungsleitfaden-ISiK-Basismodul-Stufe-3/markdown-UebergreifendeFestlegungen-UebergreifendeFestlegungen-Rest?version=current